images/virtualization-artifact/Taskfile.dist.yaml

```diff
@@ -682,6 +658,22 @@ tasks:
       - _ensure:mirrord
     cmd: ./hack/mirrord.sh wipe --deployment=virtualization-api --namespace=d8-virtualization
 
+  generate:
+    desc: "Regenerate all"
+    cmd: task api:generate
+
+  generate:core:
+    desc: "Regenerate code for core components."
+    cmd: task api:generate:core
+
+  generate:subresources:
+    desc: "Regenerate code for subresources components"
+    cmd: task api:generate:subresources
+
+  ci:generate:
+    desc: "Run generations and check git diff to ensure all files are committed"
+    cmd: task api:ci:generate
+
```

Comment:
I think these shortcuts are not needed. task list will show all api:NNNNN tasks.